fecgov/fec-cms
# Add connected organization and organization type for SSFs to the committee about tab

## Summary

- Resolves #3078 
- Resolves #3079  

Adds connected organization for organization types:

- C corporation
- L labor organization
- M membership organization
- T trade association
- V cooperative
- W corporation without capital stock

Adds organization type to the "Committee type" field if the attribute exists. The existence of organization type classifies the committee as an SSF.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Committee profile page - About this committee tab

## Screenshots
Example: http://localhost:8000/data/committee/C00702092/?tab=about-committee
![Screen Shot 2019-09-12 at 3 43 56 PM](https://user-images.githubusercontent.com/12799132/64816235-45c5ed80-d575-11e9-9ff9-e9ecda770996.png)

## How to test
- ./manage.py runserver
- Check any organization that is the following organization_type: C, L, M, T, V, and W has a connected committee on the page. Examples: 

org_tp | committee_id
-- | --
W | [C00690149](http://localhost:8000/data/committee/C00690149/?tab=about-committee)
C | [C00709956](http://localhost:8000/data/committee/C00709956/?tab=about-committee)
L | [C00702043](http://localhost:8000/data/committee/C00702043/?tab=about-committee)
V | [C00681338](http://localhost:8000/data/committee/C00681338/?tab=about-committee)
M | [C99002727](http://localhost:8000/data/committee/C99002727/?tab=about-committee)
T | [C00702092](http://localhost:8000/data/committee/C00702092/?tab=about-committee)

- Check other committees that are not the above org type. Such as candidate committees: [C00431445](http://localhost:8000/data/committee/C00431445/?tab=about-committee)
- Check that organization type is listed on the page if it exists. Example: [C00690149](http://localhost:8000/data/committee/C00690149/?tab=about-committee)
